omnigibson/robots/articulated_trunk_robot.py

```diff
@@ -0,0 +1,184 @@
+from abc import abstractmethod
+
+import torch as th
+
+from omnigibson.robots.manipulation_robot import ManipulationRobot
+from omnigibson.utils.python_utils import classproperty
+
+
+class ArticulatedTrunkRobot(ManipulationRobot):
+    """
+    ManipulationRobot that is is equipped with an articulated trunk.
```

Comment:
nit: is this always true? couldn't this be independent of a manipulation robot?

I suppose we do it in this way because we have logic that specifically ties the arm control with the trunk control. Is that correct?